lib/pack.js

```diff
@@ -3,12 +3,31 @@
 const BbPromise = require('bluebird');
 const program = require('child_process');
 
+function getLambdaPackageFlags(service) {
+
+  let myFramework = '';
+  var frameworkList = {
+    "dotnet6": "net6.0" //can add more mappings for more support 
+  };
+  if (service.provider.runtime && frameworkList[service.provider.runtime]) {
+    
+    myFramework = ' -f ' + frameworkList[service.provider.runtime];
+    if (service.provider.runtime == "dotnetcore3.1") { //dotnet 3.1 doesn't need extra parameters
```

Comment:
This condition looks like it never would be true, I think you can remove it

Reply:
removed
` if (service.provider.runtime == "dotnetcore3.1") {`
this is now verified by 
` if (service.provider.runtime && frameworkList[service.provider.runtime]) {`